easybuild/easyconfigs/p/PhyloPhlAn/PhyloPhlAn-3.0.2-foss-2021a.eb

```diff
@@ -0,0 +1,55 @@
+# Contribution by
+# DeepThought, Flinders University
+# R.QIAO <rob.qiao@flinders.edu.au>
+# update to v3.0.2
+
+easyblock = 'PythonPackage'
+
+name = 'PhyloPhlAn'
+version = '3.0.2'
+
+homepage = 'https://github.com/biobakery/phylophlan'
+description = """
+PhyloPhlAn is an integrated pipeline for large-scale phylogenetic profiling of
+genomes and metagenomes. PhyloPhlAn is an accurate, rapid, and easy-to-use method
+for large-scale microbial genome characterization and phylogenetic analysis
+at multiple levels of resolution.
+"""
+
+source_urls = ['https://github.com/biobakery/%(namelower)s/archive/refs/tags']
+sources = ['%(version)s.tar.gz']
+checksums = ['c342116662bbfbb49f0665291fc7c0be5a0d04a02a7be2da81de0322eb2256b4']
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('Biopython', '1.79'),
+    ('DendroPy', '4.5.2'),
+    ('matplotlib', '3.4.2'),
+    ('Seaborn', '0.11.2'),
+    ('FastTree', '2.1.11'),
+    ('IQ-TREE', '2.1.3'),
+    ('MUSCLE', '5.0.1428', '', True),
```

Comment:
@manifestoso the dependency on MUSCLE should be downgraded to 3.8.1551. MUSCLE v5 is a recent major re-write that changes the syntax of several commands and PhyloPhlAn has not been updated since its release.

Reply:
Thx @lexming. A good point here, I thought only elementary `MUSCLE` functions are used in alignment, but you are right, we should avoid stretching the scope. 👍